UnityProject/Assets/Scripts/Items/Weapons/GunElectrical.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+﻿using System.Collections;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6327776)